test/suites/test_push.py

```diff
@@ -0,0 +1,257 @@
+                'kwargs': {
+                    'op_list': [],
+                    'on_push': push_callback,
+                    # on_push_ctx must be set manually when running the test.
+                    'on_push_ctx': None,
+                }
+            },
+            'output': {
+                'callback_res': [[[100, 1]], [[200, 1]], [[300, 1]]],
+                # resp not used in the test output.
+                'resp': None,
+            },
+        },
+        'delete': {
+            'input': {
+                'args': ['tester', 1],
+                'kwargs': {
+                    'on_push': push_callback,
+                    # on_push_ctx must be set manually when running the test.
+                    'on_push_ctx': None,
+                }
+            },
+            'output': {
+                'callback_res': [[[100, 1]], [[200, 1]], [[300, 1]]],
+                'resp': [1, 'Bill'],
+            },
+        },
+    }
+
+    def test_00_00_push_via_connection(self):
+        for case_name in self.push_test_cases.keys():
+            with self.subTest(name=case_name):
+                callback_res = []
+                case = self.push_test_cases[case_name]
+                testing_function = getattr(self.conn, case_name)
+                case['input']['kwargs']['on_push_ctx'] = callback_res
+                resp = testing_function(
+                    *case['input']['args'],
+                    **case['input']['kwargs']
+                )
+                self.assertEqual(callback_res, case['output']['callback_res'])
+                if len(resp.data) > 0:
+                    self.assertEqual(resp.data[0], case['output']['resp'])
+
+    def test_00_01_push_via_mesh_connection(self):
+        for case_name in self.push_test_cases.keys():
+            with self.subTest(name=case_name):
+                callback_res = []
+                case = self.push_test_cases[case_name]
+                testing_function = getattr(self.mesh_conn, case_name)
+                case['input']['kwargs']['on_push_ctx'] = callback_res
+                resp = testing_function(
+                    *case['input']['args'],
+                    **case['input']['kwargs']
+                )
+                self.assertEqual(callback_res, case['output']['callback_res'])
+                if len(resp.data) > 0:
+                    self.assertEqual(resp.data[0], case['output']['resp'])
+
+    def test_00_02_push_via_connection_pool(self):
+        for case_name in self.push_test_cases.keys():
+            with self.subTest(name=case_name):
+                callback_res = []
+                case = self.push_test_cases[case_name]
+                testing_function = getattr(self.conn_pool, case_name)
+                case['input']['kwargs']['on_push_ctx'] = callback_res
+                resp = testing_function(
+                    *case['input']['args'],
+                    **case['input']['kwargs'],
+                    mode=tarantool.Mode.ANY
```

Comment:
Let's set `RW` here. If one would change the connection pool to have more than one instance, this call would fail on insert/delete/etc with `ALL`

Reply:
Done, now it is `RW`